src/Lib/Core/Jwt.hs

```diff
@@ -37,11 +38,11 @@ jwtPayloadFromMap claimsMap = do
   jwtUserId <- UUID.fromText jwtId
   return JWTPayload{..}
 
-mkJWTToken :: (MonadIO m, MonadReader AppEnv m) => Int -> JWTPayload -> m Text
+mkJWTToken :: (MonadIO m, MonadReader AppEnv m) => Seconds -> JWTPayload -> m Text
 mkJWTToken expiryInSeconds payload = do
```

Comment:
It's better to pattern-match on `expiryInSecods` like `(Seconds expiry)`. Note, that you don't need now `*Seconds` suffix, because `newtype` explicitly says what you might want to expect.